index.js

```diff
@@ -80,8 +80,9 @@ async function processReport(token, workSpace, plugins, currentRunnerID) {
         return
     }
 
+    let issueTitle = core.getInput('issue_title');
```

Comment:
Can we move this to the main method (run) and pass as a function parameter (processReport). Because we are trying to move common code to a NPM package.